File: Kth_Largest_Element_in_an_Array.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
const findKthLargest = function(nums, k) {
  //   sort array
  //     slice from back to find the kth value
  nums.sort((first, second) => second - first);
  console.log(nums);
  return nums[k - 1];
};

File: Leetcode_solutions/Kth Largest Element in an Array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        if k > len(nums) or len(nums) == 0:
            return -1
        nums.sort()
        return nums[-k]


class Solution:
    def findKthLargest(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        return heapq.nlargest(k, nums)[-1]
